src/org/zaproxy/zap/extension/pscanrulesBeta/CookieLooselyScopedScanner.java

```diff
@@ -140,8 +140,8 @@ private void raiseAlert(HttpMessage msg, int id, String host, List<HttpCookie> l
 				Constant.messages.getString(MESSAGE_PREFIX + "extrainfo", host, sbCookies),
 				getSolutionMessage(), getReferenceMessage(), 
 				"",	// No Evidence
-				0,	// TODO CWE Id
-				0,	// TODO WASC Id
+				784, // CWE-784: Reliance on Cookies without Validation and Integrity Checking in a Security Decision

```

Comment:
Might be too specific? We are not checking which "type" of cookies are being flagged (it could be a language or theme cookie, not related to a session or other privileges).